src/QueryBuilder/ColumnConverter.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\BigQuery\QueryBuilder;
+
+use Keboola\Datatype\Definition\Bigquery;
+use Keboola\StorageDriver\Command\Table\ImportExportShared\DataType;
+use Keboola\TableBackendUtils\Escaping\Bigquery\BigqueryQuote;
+
+class ColumnConverter
+{
+    public const DATA_TYPES_OPTIONS = [
+        DataType::INTEGER,
+        DataType::REAL,
+    ];
+
+    public const DATA_TYPES_MAP = [
+        DataType::STRING => Bigquery::TYPE_STRING,
+        DataType::INTEGER => Bigquery::TYPE_INT64,
+        DataType::DOUBLE => Bigquery::TYPE_NUMERIC,
+        DataType::BIGINT => Bigquery::TYPE_BIGINT,
+        DataType::REAL => Bigquery::TYPE_NUMERIC,
```

Comment:
@zajca skontroloval by som este to mapovanie, ked som to zacal implementovat tak som tam dal len nieco aby to cca sedelo ale chcel som to este preverit, pak finalne